tests/README.md

```diff
@@ -269,9 +269,31 @@ The Test Suite will pause its execution, and you will be able to SSH into the en
 
 #### How rule validation scenarios work
 
-In directory `data` are directories mirroring `group/group/.../rule`
-structure of datastream. In reality, only name of the rule directory needs to be
-correct, group structure is currently not used.
+In directory `/tests/data` there are directories mirroring tree structure of
```

Comment:
This readme is already in `tests` directory.

Reply:
I didn't mean for my comment to be added to the file.

The doc says: ```In directory `/tests/data` there are directories ...```.
But when reading the `README.md` file, the person is already in `tests directory`, so I suggest:
```In directory `data/` there are directories ...```.